rich/logging.py

```diff
@@ -41,9 +41,10 @@ class RichHandler(Handler):
             Defaults to 10.
         locals_max_string (int, optional): Maximum length of string before truncating, or None to disable. Defaults to 80.
         log_time_format (Union[str, TimeFormatterCallable], optional): If ``log_time`` is enabled, either string for strftime or callable that formats the time. Defaults to "[%x %X] ".
+        keywords (List[str], optional): List of words to highlight instead of ``RichHandler.KEYWORDS``.
     """
 
-    KEYWORDS: ClassVar[Optional[List[str]]] = [
+    KEYWORDS: ClassVar[List[str]] = [
```

Comment:
The typing can't be changed now. Since its been optional since day one, people may have reasonably set it to None for no keywords.